src/button/button-group.scss

```diff
@@ -61,28 +55,35 @@
           }
         }
       }
+    }
 
-      // Reset border radius only for non-borderless items
-      &:not(.iui-borderless) {
-        border-radius: 0;
-      }
+    // #region Borders
+    $childSelector: 'input, .iui-button';
```

Comment:
Why one is targeting base element and other our class?

Reply:
i changed `button` to `.iui-button` because it can be rendered as a link

Reply:
So don't we want to be consistent and have CSS selectors here instead?

Reply:
What do you mean by "CSS selectors"?

Reply:
Ups, wanted to say classes

Reply:
Updated the selectors, take a look